src/utils/check-is-vm.ts

```diff
@@ -0,0 +1,55 @@
+import { exec } from 'child_process';
+import os from 'os';
+
+const WIN_PLATFORM   = 'win32';
+const LINUX_PLATFORM = 'linux';
+const MAC_PLATFORM   = 'darwin';
+const VM_REGEX = /virtual|vmware|hyperv|wsl|hyper-v|microsoft|parallels|qemu/gi;
+
+function getCommandOutput (command: string): Promise<string> {
+    return new Promise(resolve => {
+        exec(command, (error, stdout, stderr) => resolve(stdout));
+    });
+}
+
+function isLinuxVM (): Promise<boolean> {
+    const LINUX_COMMAND   = 'systemd-detect-virt';
+    const NOT_FOUND_REGEX = /systemd-detect-virt: not found/ig;
+
+    return new Promise(resolve => {
+        let error = '';
+
+        exec(LINUX_COMMAND, (err, stdout, stderr) => {
```

Comment:
Use promisified `exec` from utils and get rid of Promise

https://github.com/DevExpress/testcafe/blob/5b776fe8b211599a2911eed84d5809ab3c4fa833/src/utils/promisified-functions.js#L13